ui/components/resolvers/ResolverDetail.js

```diff
@@ -182,9 +192,20 @@ class ResolverDetail extends Component {
                     >
                         Save
                     </Button>
-                    {/* <Button bsStyle="default" onClick={() => console.log("cancel")}>Cancel</Button> */}
+                    <Button
```

Comment:
This button needs to be disabled when delete is not possible (i.e. before saving a new resolver)